geomesa-tools/bin/common-functions.sh

```diff
@@ -96,6 +98,20 @@ function containsElement() {
   return 1
 }
 
+function registerAutocomplete() {
+  eval compFile="~/.bash_completion" # resolve tilde
+  [[ -f ${compFile} ]] || touch ${compFile}
+  # Search .bash_completion for this entry so we don't add it twice
+  head=$(head -n 1 ${GEOMESA_CONF_DIR}/autocomplete.sh)
+  res=$(grep -F $head ${compFile})
+  if [[ -z "${res}" ]]; then
+    echo "Installing Autocomplete Function"
+    cat ${GEOMESA_CONF_DIR}/autocomplete.sh >> ${compFile}
```

Comment:
Note to self: Probably good to check for errors that could happen here (access denied) and suppress them since if this fails then it will retry on every run of the tools.